### ISSUE TITLE
First Contentful Paint metric is stored in a different json object depending on the host app (Chrome vs Gecko)

### ISSUE THREAD

**Issue by acreskeyMoz** [mozilla/browsertime#39]
Looking at the `browsertime.json` for Gecko we find First Contentful Paint here:
`[0].statistics.timings.timeToContentfulPaint`

And in Chrome it's output here:
`[0].statistics.timings.paintTiming.first-contentful-paint`

This is problematic for performance tooling.

Even if the user agents implement this paint timing metric in a different manner, I feel that browsertime should be consistent in its output.

**Comment by acreskeyMoz** [comment]
I can take this one in a bit. If the change should be made upstream, I can make it there and bring it back to this branch.

**Comment by ncalexan** [comment]
> I can take this one in a bit. If the change should be made upstream, I can make it there and bring it back to this branch.

I agree that Browsertime should be consistent across UAs, and agree that this should be addressed upstream (if it hasn't already been addressed).

Just FYI, I merged in upstream (https://github.com/ncalexan/browsertime/tree/v6.0.0-alpha.5-android) but haven't pushed to `m/b` 'cuz Geckodriver + Android is still outstanding.  So there's hope for "just" fixing it upstream in the near future.

**Comment by gmierz** [comment]
Another metric Chrome browsertime data is missing is `timeToDomContentFlushed`.
